scene/2d/remote_transform_2d.cpp

```diff
@@ -63,7 +63,21 @@ void RemoteTransform2D::_update_remote() {
 		return;
 
 	//todo make faster
-	n->set_global_transform(get_global_transform());
+	if (use_global_coordinates) {
+		if (update_remote_position)
+			n->set_global_position(get_global_position());
+		if (update_remote_rotation)
+			n->set_global_rotation(get_global_rotation());
+		if (update_remote_scale)
+			n->set_global_scale(get_global_scale() + Vector2(1, 1));
```

Comment:
I'll make changes so that set_global_transform is used if all options are true. I forgot about transforms like shears, whoops!

On using global transform for rotation and scale, won't switching the origin to (0, 0) cause issues? Then the remote node's position will be set to (0, 0), right? I could get the node's global transform and change origin to the node's origin. That might be the way to do it. I'll do some testing and see what happens!